site/_layouts/article.njk

```diff
@@ -19,10 +19,8 @@ layout: _wrapper
 
   <h1 class="article__header type--h2">{{title}}</h1>
 
-  <aside class="article__info">{% if section == 'posts' %}{{ dateTime(microcopy.datePosted, page.date, locale.code, options={ year: 'numeric', month: 'long', day: 'numeric' }) }}{{ authorsList(authors, contributors, microcopy) }}{% elif section == 'case-studies' %}{{ appInfo(app) }}{% endif %}</aside>
-
-  {% if content | toc %}
-  <aside class="article__toc">{{ tableOfContents(content, microcopy.toc.title, {title: microcopy.skip.toc, id: 'article'}) }}</aside>
+  {% if content | toc or authors %}
+  <aside class="article__toc">{% if section == 'posts' %} {{ authorsList(microcopy.posted, authors, contributors, microcopy) }} {{ dateTime(microcopy.datePosted, page.date, locale.code, options={ year: 'numeric', month: 'long', day: 'numeric' }) }} {% elif section == 'case-studies' %} {{ appInfo(app) }} {% endif %} {{ tableOfContents(content, microcopy.toc.title, {title: microcopy.skip.toc, id: 'article'}) }}</aside>
```

Comment:
I think that now that this has a couple of items in the `aside`, it needs `sections` for each, and each `section` should have an associated header (probably an `h4`?) instead of the `span` that you have above and the `p` that's currently there for Date Posted. In fact, I may go so far as to say that the author and date posted bits belong in a [`footer`](http://html5doctor.com/the-footer-element-update/) tag, with only the table of contents belonging in an [`aside`](http://html5doctor.com/aside-revisited/).

Reply:
The suggestion here for author/date information to be placed in a footer tag outside of the <aside> will affect the current grid. I would like to wait until I have worked on the new design for the News article to make these changes.

Reply:
I'm OK with that. Can you add a reminder to that issue to include this work so it doesn't get lost?

Thanks!